packages/presentation/lib/screen/home/home_bloc.dart

```diff
@@ -5,7 +5,7 @@ import 'package:domain/enum/movie_type.dart';
 import 'package:domain/model/movie_response.dart';
 import 'package:domain/use_case/get_movies_use_case.dart';
 import 'package:presentation/base/bloc.dart';
-import 'package:presentation/screen/home/enum/tab_state.dart';
+import 'package:presentation/enum%20/tab_state.dart';
```

Comment:
seems that enum folder has space in the end. remove it

Reply:
removed